solidity/contracts/TokenStakingEscrow.sol

```diff
@@ -247,6 +269,19 @@ contract TokenStakingEscrow is Ownable {
         emit DepositWithdrawn(operator, grantee, amount);
     }
 
+    function withdrawRevoked(
+        Deposit memory deposit,
+        address operator,
+        address grantManager
+    ) internal {
+        uint256 amount = deposit.amount - deposit.withdrawn;
+
+        deposits[operator].withdrawn = deposit.amount;
```

Comment:
Can revocation be performed in a couple of turns with different amounts or is it happening just once for total amount of tokens?

Reply:
Grant once revoked cannot be revoked again and grant one revoked cannot be un-revoked.